LICENSE

```diff
@@ -11,7 +12,8 @@ modification, are permitted provided that the following conditions are met:
       this list of conditions and the following disclaimer in the documentation
       and/or other materials provided with the distribution.
 
-    * Neither the name of the SKA South Africa project nor the names of its
+    * Neither the name of the National Research Foundation
+      (South African Radio Astronomy Observatory) project nor the names of its
```

Comment:
I would rather say something like:

`Neither the name of the National Research Foundation or South African Radio Astronomy Observatory nor the names...`